Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -202,6 +202,7 @@ class Document
     bool in_quirks_mode() const { return m_quirks_mode == QuirksMode::Yes; }
     void set_quirks_mode(QuirksMode mode) { m_quirks_mode = mode; }
 
+    ExceptionOr<NonnullRefPtr<Node>> import_node(NonnullRefPtr<Node>, bool);
```

Comment:
Please don't omit the parater name in a declaration if its meaning is not clear from context, like here

Reply:
Done :+1: